guides/doc-Provisioning_Guide/topics/Resources.adoc

```diff
@@ -1,16 +1,7 @@
 [[Configuring_Provisioning_Resources]]
 == Configuring Provisioning Resources
 
-{ProjectNameX} contains provisioning resources that you must setup and configure to create a host. These include:
-
-Domain::
-You must assign every host that is managed by {Project} to a domain. Using the domain, {Project} can manage A, AAAA and PTR records. If there is no record integration, you still must create and associate at least one domain. Domains are included in the naming conventions {Project} hosts, for example, a host `test123` in the `example.com` domain has the host name `test123.example.com`.
-
-Subnet::
-You must assign every host managed by {Project} to a subnet. Using subnets, {Project} can then manage IPv4 reservations. If  there are no reservation integrations, you still must create and associate at least one subnet. You can manage IP addresses with one of the following options:
-* Manual - when the *IPAM* is set to *None*
-* DHCP - the managed DHCP server requests a free IP from its pool
-* Database - serial or random IP address book that is kept in the {Project} database
+{Project} contains provisioning resources that you must set up and configure to create a host. These include:
```

Comment:
Sounds like this information should be under `=== Provisioning Resources Overview`. It does not belong under the COnfiguring chapter directly because it has no action items here.

Reply:
I disagree with you here, @spetrosi this is just an introduction to the resources that you'll deal with in Satellite provisioning.